addons/web/static/src/webclient/actions/action_service.js

```diff
@@ -857,7 +857,9 @@ function makeActionManager(env) {
                     type: "warning",
                 });
             }
-            if (options.onClose) {
+            if (action.close_on_report_download) {
```

Comment:
`close_on_report_download` makes no sense in this context, I'd simply rename into `close`